src/ext/java/org/opentripplanner/ext/gtfsgraphqlapi/generated/GraphQLTypes.java

```diff
@@ -2743,7 +2743,6 @@ public enum GraphQLRoutingErrorCode {
     NO_TRANSIT_CONNECTION_IN_SEARCH_WINDOW,
     OUTSIDE_BOUNDS,
     OUTSIDE_SERVICE_PERIOD,
-    SYSTEM_ERROR,
```

Comment:
If this wasn't an accident (it's a generated file) then you also need to remove it from `schema.graphqls`